students/dowoon/users/views.py

```diff
@@ -35,3 +35,22 @@ def post(self, request):
         except ValidationError as e:
             return JsonResponse({"message" : e.message}, status=400)
 
+class LoginView(View):
+    def get(self, request):
+        data = json.loads(request.body)
+
+        try:
+            email    = data['email']
+            password = data['password']
+        except KeyError:
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
+
+        try:
+            user = User.objects.get(email=email)
+        except User.DoesNotExist:
+            return JsonResponse({"message": "INVALID_USER (email)"}, status=401)
+
+        if password != user.password:
+            return JsonResponse({"message": "INVALID_USER (password)"}, status=401)
+        else:
+            return JsonResponse({"message": "SUCCESS"}, status=200)
```

Comment:
굳이 `else`가 필요한게 아니기 때문에 지워주시는게 좋습니다!
```suggestion
        if password != user.password:
            return JsonResponse({"message": "INVALID_USER (password)"}, status=401)
            
        return JsonResponse({"message": "SUCCESS"}, status=200)
```

Reply:
확인 / 수정했습니다